src/org/opensolaris/opengrok/analysis/FileAnalyzer.java

```diff
@@ -211,34 +232,36 @@ public void analyze(Document doc, StreamSource src, Writer xrefOut)
     }
 
     /**
-     * Derived classes should override to write a cross referenced HTML file
-     * for the specified args.
+     * Derived classes should override to write a cross referenced HTML file for
+     * the specified args.
+     *
      * @param args a defined instance
      * @return the instance used to write the cross-referencing
      * @throws java.io.IOException if an error occurs
      */
     public Xrefer writeXref(WriteXrefArgs args) throws IOException {
         throw new UnsupportedOperationException(
-            "Base FileAnalyzer cannot write xref");
+                "Base FileAnalyzer cannot write xref");
     }
-    
-    protected final JFlexTokenizer symbolTokenizer;
-    
+
+    // you analyzer HAS to override this to get proper symbols in results
+    protected JFlexTokenizer symbolTokenizer;
```

Comment:
This may be an accidental regression from `final`.

Reply:
yes, sorry, reverting

Reply:
(and good catch, thank you!)